certified-connectors/DocuSignDemo/script.csx

```diff
@@ -2063,11 +2063,6 @@ public class Script : ScriptBase
       var content = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
       var body = ParseContentAsJObject(content, false);
 
```

Comment:
I am removing this block of code to test it on the Preview environment on the issue of creating duplicate configs in the Connect.